hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/metadata/WitnessedContainerDBDefinition.java

```diff
@@ -62,7 +61,7 @@ public String getLocationConfigKey() {
     return ScmConfigKeys.OZONE_SCM_DATANODE_ID_DIR;
   }
 
-  DBColumnFamilyDefinition<ContainerID, String> getContainerIdsTable() {
+  DBColumnFamilyDefinition<ContainerID, ContainerCreateInfo> getContainerIdsTable() {
```

Comment:
This constructor can initialize from the map of Definitions.
```suggestion
  DBColumnFamilyDefinition<ContainerID, ContainerCreateInfo> getContainerIdsTable() {
      super(COLUMN_FAMILIES);
  }
```